### ISSUE TITLE
[req]: Torrent Heaven (Dutch)

### ISSUE THREAD

**Issue by tyhawk** [Jackett/Jackett#14852]
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Private

### Tracker details

```markdown
- Name: Torrent Heaven
- URL: https://www.torrentheaven.org/
- Description: Torrent Heaven is a DUTCH Private Torrent Tracker for MOVIES / TV / GENERAL
- Minimum Seeding Time and/or Minimum Seeding Ratio (if Private): 1.00 ratio minimum / 1.5 maximum ratio
- Additional Comments:
There is an indexer with the same name, but that one is German based.
similar to muziekfabriek
signups open
```

**Comment by garfield69** [comment]
v0.21.1230

**Comment by garfield69** [comment]
https://www.torrentheaven.org/ cloudflare Web server is returning an unknown error Error code 520

**Comment by tyhawk** [comment]
It's already back up. 520 is a server error, not a 404 or 410. It seems they solved the issue, whatever it was.

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.torrentheaven.org/ > Oops! Something went wrong... If this problem persists, please contact an administrator and report the following code: #ERROR602244

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.torrentheaven.org/ >
```
Unfortunately there is a cyber attack at the moment, we are working on other hosting.!
We would also like to know the reason for this Cyber ​​attack, you can post this on duken.nl
```

**Comment by garfield69** [comment]
online.

**Comment by garfield69** [comment]
- https://www.torrentheaven.org/ > Oops! Something went wrong... If this problem persists, please contact an administrator and report the following code: ?ERROR602163
- Not Found   The requested URL was not found on this server.   Apache/2.4.52 (Ubuntu) Server at www.torrentheaven.org Port 80
- homepage online but there are no torrents
- 5 sec retry loop with Klein momentje
-  Ubuntu Apache2 Default Page

**Comment by joseperk-byte** [comment]
ik heb hetzelfde probleem, is de site down?

**Comment by garfield69** [comment]
Site appears to be back online gain, however it seems to be a complete reset.
After registering, I tried to login but received:
```
Oeps! Er is iets misgegaan... Als dit probleem blijft bestaan, neem dan contact op met een administrator en meldt de volgende code: #ERROR602283
```

**Comment by ilike2burnthing** [comment]
Appears to be back up, I got straight into my old account. If there's any difficulties, try a private window.

**Comment by stadiaoffred** [comment]
Does this one still exist? Can't seem to find it under indexers.

**Comment by ilike2burnthing** [comment]
Yes. If it's not under your configured indexers then go to `Add indexer` and find it there.

**Comment by stadiaoffred** [comment]
That's what I mean, I can't find it under list in "add indexer".

I'm using Version v0.24.170, if that matters.

**Comment by ilike2burnthing** [comment]
Current release is v0.24.222, but that shouldn't really matter as the indexer was added in 2023, had two minor commits in 2024, and has remained unchanged since then - https://github.com/Jackett/Jackett/commits/master/src/Jackett.Common/Definitions/torrent-heaven.yml

If the indexer isn't configured and isn't available to add, then your install is broken and you should open a new issue.

**Comment by garfield69** [comment]
https://www.torrentheaven.org/ > redirecting https://router.parklogic.com/

**Comment by garfield69** [comment]
online
